src/Nethermind/Nethermind.Evm/CodeAnalysis/CMSketch.cs

```diff
@@ -0,0 +1,83 @@
+
+using System;
+using System.Collections.Concurrent;
+
+namespace Nethermind.Evm.CodeAnalysis
+{
+
+    public class CMSketch
+    {
+        private ConcurrentDictionary<(int Row, int Column), uint> _sketch;
```

Comment:
If we have a `row` and `column`, cannot it be represented as a single array with new `uint[rows * columns]`? Does it require a sparse representation?

Reply:
TODO